index.js

```diff
@@ -1023,7 +1022,8 @@ module.exports = function (log, indexesPath) {
         timestamp: indexes['timestamp'].tarr[seq],
       })
     })
-    sortedCache[order].set(bitset, fpq.clone())
+    fpq.trim()
+    sortedCache[order].set(bitset, fpq)
```

Comment:
On the other hand, I think this clone was not needed.

And I added the trim() because FPQ docs said 

>trim(): clean up unused memory in the heap; useful after high-churn operations like many add()s then remove()s.